src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,32 @@
+package calculator;
+
+import org.apache.logging.log4j.util.Strings;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+public class StringCalculator {
+    private final int DEFAULT_INT_VALUE = 0;
+
+    public StringCalculator(){
```

Comment:
IntelliJ IDEA의 코드 자동 정렬 기능(⌥⌘L, Ctrl+Alt+L)을 사용하면 더 깔끔한 코드를 볼 수 있을 거예요.

```suggestion
    public StringCalculator() {
```